docs/tutorials/task.csv-to-oscal-cd/csv-to-oscal-cd.md

```diff
@@ -0,0 +1,151 @@
+
+An example input [csv-file](ocp4-sample-input.csv).
+
+An example output [json-file](component-definition.json).
+
+<table>
+
+<tr style="text-align:left;vertical-align:top">
+<th>Column Name
+<th>Value Type
+<th>Specification
+<th>Value Description
+<th>Component Definition Locale
+<th>Example Value
+
+<tr style="text-align:left;vertical-align:top">
+<td>Rule_Id
+<td>String
+<td>required
+<td>A textual label that uniquely identifies a policy (desired state) that can be used to reference it elsewhere in this or other documents.
+<td>component.control-implementation.prop["Rule_Id"]
+<td>password_policy_min_length_characters
+
+<tr style="text-align:left;vertical-align:top">
+<td>Rule_Description
+<td>String
+<td>required
+<td>A description of the policy (desired state) including information about its purpose and scope.
+<td>component.control-implementation.prop["Rule_Description"]
+<td>Ensure password policy requires minimum length of 12 characters
+
+<tr style="text-align:left;vertical-align:top">
+<td>Profile_Reference_URL
+<td>String
+<td>required
+<td>A URL reference to the source catalog or profile for which this component is implementing controls for. A profile designates a selection and configuration of controls from one or more catalogs
+<td>component.control-implementation.source
+<td>https://github.com/usnistgov/oscal-content/blob/main/nist.gov/SP800-53/rev5/json/NIST_SP-800-53_rev5_HIGH-baseline_profile.json
+
+<tr style="text-align:left;vertical-align:top">
+<td>Profile_Description
+<td>String
+<td>required
+<td>A description of the profile.
+<td>component.control-implementation.description
+<td>NIST Special Publication 800-53 Revision 5 HIGH IMPACT BASELINE
+
+<tr style="text-align:left;vertical-align:top">
+<td>Component_Type
+<td>String
+<td>required
+<td>A category describing the purpose of the component.
+<td>component.type
+<td>Validation
+
+<tr style="text-align:left;vertical-align:top">
+<td>Control_Mappings
+<td>String List (blank separated)
+<td>required
+<td>A list of textual labels that uniquely identify the controls or statements that the component implements.
+<td>component.control-implementation.implemented-requirement.statement.statement-id<br>*and*<br>component.control-implementation.implemented-requirement.control-id
+<td>ia-5.1_smt.a ia-5.1
+
+<tr style="text-align:left;vertical-align:top">
+<td>Resource
+<td>String
+<td>required
+<td>A human readable name for the component.
+<td>component.title
+<td>Compliance Center
+
+<tr style="text-align:left;vertical-align:top">
+<td>Parameter_Id
+<td>String
+<td>optional
+<td>A textual label that uniquely identifies the parameter associated with that policy (desired state) or controls implemented by the policy (desired state).	A description of the parameter including the purpose and use of the parameter.
+<td>component.control-implementation.prop["Parameter_Id"]<br>*and*<br>component.control-implementation.set-parameter.param-id
+<td>minimum_password_length
+
```

Comment:
policy rule instead of policy
"or controls implemented by the policy (desired state)." this should be removed.
"	A description of the parameter including the purpose and use of the parameter." this should be removed as it is about param description.